packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -146,53 +153,56 @@ export class CustomScrollbar extends Component<Props> {
   }
 }
 
-export default CustomScrollbar;
-
-const getStyles = (isPageScrollbar?: boolean): StylesInterface => {
-  return {
-    customScrollbar: css`
-      display: flex;
-      flex-grow: 1;
-      &:hover {
-        opacity: 0.8;
-        transition: opacity 0.3s ease-in-out;
-      }
-    `,
-    trackVertical: css`
-      border-radius: 3px;
-      width: 8px !important;
-      right: ${isPageScrollbar ? 0 : 2}px;
-      bottom: 2px;
-    `,
-    trackHorizontal: css`
-      border-radius: 3px;
-      height: 8px !important;
-      right: 2px;
-      bottom: 2px;
-      left: 2px;
-    `,
-    thumbVertical: css`
-      @include linear-gradient(#404357, #424345);
-      border-radius: 6px;
-      opacity: 0;
-      &:hover {
-        opacity: 0.8;
-        transition: opacity 0.3s ease-in-out;
-      }
-    `,
-    thumbHorizontal: css`
-      @include gradient-horizontal(#404357, #404357);
-      border-radius: 6px;
-      opacity: 0;
-      &:hover {
-        opacity: 0.8;
-        transition: opacity 0.3s ease-in-out;
-      }
-    `,
-    view: css`
-      display: flex;
-      flex-grow: 1;
-      flex-direction: column;
-    `,
-  };
-};
+export const CustomScrollbar = withTheme(UnthemedCustomScrollbar);
+CustomScrollbar.displayName = 'CustomScrollbar';
+
+const getStyles = stylesFactory(
+  (theme: GrafanaTheme, isPageScrollbar?: boolean): StylesInterface => {
+    return {
+      customScrollbar: css`
+        display: flex;
+        flex-grow: 1;
+        &:hover {
+          opacity: 0.8;
+          transition: opacity 0.3s ease-in-out;
+        }
+      `,
+      trackVertical: css`
+        border-radius: 3px;
+        width: 8px !important;
+        right: ${isPageScrollbar ? 0 : 2}px;
+        bottom: 2px;
+      `,
+      trackHorizontal: css`
+        border-radius: 3px;
+        height: 8px !important;
+        right: 2px;
+        bottom: 2px;
+        left: 2px;
```

Comment:
nit: Maybe see if you can find any spacing in the theme that matches these `px` values?